examples/demo/worker/services/worker.go

```diff
@@ -15,18 +15,15 @@ type Worker struct {
 }
 
 // Configure starts workers and register rpc job
-func (w *Worker) Configure() error {
-	err := pitaya.StartWorker(pitaya.GetConfig())
-	if err != nil {
-		return err
-	}
-
-	pitaya.RegisterRPCJob(&RPCJob{})
-	return nil
+func (w *Worker) Configure(app pitaya.Pitaya) {
+	app.StartWorker()
```

Comment:
Because the error could be generated by config parsing. As it's not being configured here, there's no more errors that could occur in this place. Now, it's being handled in the worker constructor.
This is the old snippet:
```go
	hostname, err := os.Hostname()
	if err != nil {
		return nil, err
	}
```